week8/yeonwook/mission/src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import useCartStore from "../../store/useCartStore";
+import alert from "../../assets/img/alert.svg";
+import SelectedMenu from "../../components/SelectedMenu/SelectedMenu";
+import { Alert, Info, NotEnough, NotEnoughText, CartName, EnoughText, More, EnableOrder, DisabledOrder, BottomBar, MinPrice, CostDiv, LText, RText, TotalL, TotalR, BottomDiv } from "./Cart.styles";
```

Comment:
style 파일 분리해서 import 하신건 잘 하셨습니다!
```suggestion
import * as S from "./Cart.styles";
```
이런식으로 하고 사용할때는 `<S.Alert>` 이렇게 쓰면 간편하게 import 할 수 있습니다!